### ISSUE TITLE
GBA emulator - Real Time Clock (RTC) support

### ISSUE THREAD

**Issue by oliviercharavel** [jasarien/Provenance#151]
Its not high priority since only a few GBA games use Real Time Clock (RTC) but it seems like the GBA emulator doesn't support it. It would be great to have that for Pokemon hack games that support it (like Pokemon Liquid Crystal) or Boktai games (there are some IPS patches for those ones, though).

Thanks!

**Comment by jasarien** [comment]
I believe this is already supported. I tried out Pokemon Ruby, which allows you to set the clock in game at the beginning. I then left it suspended in the background (it doesn't run while in the bg, so it's impossible for it to tick away while suspended). When I resumed it some minutes later the clock was showing the correct time. See the screenshots: 
http://d.pr/i/1hnfs
http://d.pr/i/12Nnm

Now the only way I can explain this is if the emulator reads the real time from the host system and passes it through to the game.
